localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
 (function (root, factory) {
   if (typeof exports === 'object') {
     // Node.
```

Comment:
What if it the comment said "CommonJS environments"? This way browseify wouldn't be left out, and future platforms would be included.